generators/generator-base.js

```diff
@@ -1415,4 +1415,12 @@ Generator.prototype.filterNumber = function (input, isSigned, isDecimal) {
     return NaN;
 };
 
+Generator.prototype.error = function(msg) {
+    return this.env.error(chalk.red.bold('ERROR! ') + msg);
+};
+
+Generator.prototype.warning = function(msg) {
+    return this.env.error(chalk.yellow.bold('WARNING! ') + msg);
```

Comment:
This would exit the generator. Do we want to exit for warnings?  Imo we should use this.log for warning and only for error we should exit